test/api.spec.ts

```diff
@@ -1,20 +1,15 @@
 import { expect } from "chai";
```

Comment:
Is it possible to do tests for positive flow? It was there before

Reply:
done